### PR TITLE
[Tests-Only] Include tests with undefined steps in failed list in run.sh

### PR DESCRIPTION
**Pull request by saw-jan**
## Related Issue
closes https://github.com/owncloud/web/issues/5026

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15173/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15173/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15175/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15175/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15176/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15176/

<p>










</p></details>

File: tests/acceptance/run.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [ "${ACCEPTANCE_TESTS_EXIT_STATUS}" -ne 0 ]; then
   # If the second run of a scenario fails, hen it gets reported like:
   # 5) Scenario: try to login with invalid username (attempt 2) # tests/acceptance/features/webUILogin/login.feature:67
   # So only look for scenarios that failed on attempt 2.
-  FAILED_SCENARIOS="$(grep ') Scenario: .* (attempt 2)' logfile.txt)"
+  FAILED_SCENARIOS="$(grep ') Scenario: .*' logfile.txt) | grep -v '(attempt 1, retried)')"
```
Comment (kiranparajuli589) [inline tests/acceptance/run.sh:43]: just a question: are the attempt 2 failures caught after modifying this?

Comment (saw-jan) [reply tests/acceptance/run.sh:43]: first it gets all fails and filters out tests with `(attempt 1, retried)`
leaving `(attempt 2)` tests and failed-with-undefined tests in the list

File: tests/acceptance/run.sh [hunk 2]
```diff
@@ -40,7 +40,7 @@ if [ "${ACCEPTANCE_TESTS_EXIT_STATUS}" -ne 0 ]; then
   # If the second run of a scenario fails, hen it gets reported like:
   # 5) Scenario: try to login with invalid username (attempt 2) # tests/acceptance/features/webUILogin/login.feature:67
   # So only look for scenarios that failed on attempt 2.
```
Comment (Talank) [inline tests/acceptance/run.sh:42]: I think you need to update this comment as well.

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

**Review by phil-davis — APPROVED** [review]

**Pull request merged**